src/components/screens/main-screen.tsx

```diff
@@ -127,54 +147,129 @@ const MainScreen: React.FC = () => {
+      let bracketIndex = option.name.indexOf("(");
+      bracketIndex === -1 && (bracketIndex = option.name.length);
+      const cutIndex = Math.min(commaIndex, bracketIndex);
+      option.name = option.name.substring(0, cutIndex);
+      if (option.name.length > 49) {
+        option.name = `${option.name.substring(0, 49)}...`;
+      }
+    });
+
     return (
       <Stack
-        spacing={2}
-        component="form"
-        direction="row"
+        direction="column"
+        alignItems="center"
         width="100%"
+        spacing={4}
+        sx={{ padding: 2 }}
       >
+        <Typography>{strings.mainScreen.courseCodeDescription}</Typography>
         <Autocomplete
           fullWidth
           disablePortal
           options={courseOptions}
           inputValue={coursesCode}
           /* eslint-disable @typescript-eslint/no-unused-vars */
           renderOption={(props, option, _) => (<Typography noWrap {...props}>{option.name}</Typography>)}
-          onInputChange={(_, newInputValue) => setCourseCode(newInputValue)}
+          onInputChange={(_, newInputValue) => setCourseCode(newInputValue) }
+          onChange={(_, newValue) => newValue && onCourseCodeMatch(newValue.label) }
           renderInput={params =>
             <TextField
               {...params}
               variant="outlined"
               name="courseCode"
-              label={strings.mainScreen.courseCode}
+              label={strings.mainScreen.courseCodePlaceholder}
             />}
         />
-        <IconButton
-          color="primary"
-          type="submit"
-          disabled={loading}
-          onClick={() => onCourseCodeMatch(coursesCode)}
-        >
-          <Search/>
-        </IconButton>
       </Stack>
     );
   };
 
+  /**
+   * Render limitations
+   */
+  const renderLimitations = () => (
+    <Accordion sx={{ width: "100%" }}>
+      <AccordionSummary expandIcon={<Warning/>} aria-controls="limitations-content" id="limitations-header">
+        <Typography variant="h4">{strings.mainScreen.limitationsTitle}</Typography>
+      </AccordionSummary>
+      <AccordionDetails>
+        <Stack direction="column" spacing={1}>
+          {strings.mainScreen.limitations.map((limitation: string) => (
+            <Button
+              key={limitation}
+              variant="outlined"
+              disabled
+              sx={{
+                textTransform: "none"
+              }}
+            >
+              {limitation}
+            </Button>
+          ))}
+        </Stack>
+      </AccordionDetails>
+    </Accordion>
+  );
+  
+  /**
+   * Renders examples
+   */
+  const renderExamples = () => (
+    <Stack direction="column" alignItems="center" spacing={2}>
+      <Typography variant="h4">{strings.mainScreen.examplesTitle}</Typography>
+      <Stack
+        direction="row"
+        spacing={2}
+        alignItems="stretch"
+      >
+        {strings.mainScreen.examples.map((example: string) => (
```

Comment:
Except for having these example strings in localization, put them at the top of the file as e.g.

QUERY_EXAMPLES=[
  "Example1",
  "Example2",
  "Example3",
]

Because we certainly don't want to have localization for example strings